client/src/observatory/results.tsx

```diff
@@ -0,0 +1,668 @@
+                      <th>Score</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {scoringTable.map((st) => {
+                      return (
+                        <tr
+                          className={
+                            result.scan.grade === st.grade ? "current" : ""
+                          }
+                          key={st.grade}
+                        >
+                          <td>{fixMinusSymbol(st.grade)}</td>
+                          <td>
+                            {st.scoreText}{" "}
+                            {result.scan.grade === st.grade && st.stars && (
+                              <StarsSVG />
+                            )}
+                          </td>
+                        </tr>
+                      );
+                    })}
+                  </tbody>
+                </table>
+              </Tooltip>
+            </span>
+          </div>
+          {trend(result)}
+        </section>
+        <section className="data">
+          <div>
+            <a href="docs/scoring">
+              <span className="label">Score:</span>
+            </a>{" "}
+            {result.scan.score}/100
+          </div>
+          <div>
+            <a href="#scan_history">
+              <span className="label">Scan Time:</span>
+            </a>{" "}
+            {new Date(result.scan.scanned_at).toLocaleString([], {
+              dateStyle: "medium",
+              timeStyle: "medium",
+            })}
+          </div>
+          <a href="docs/scoring##tests-and-score-modifiers">
+            <span className="label">Tests Passed:</span>
+          </a>{" "}
+          {result.scan.tests_passed}/{result.scan.tests_quantity}
+        </section>
+        <section className="actions">
+          {!isServer && (
+            <CountdownButton
+              host={host}
+              from={result.scan.scanned_at}
+              duration={60}
+              title="Rescan"
+              onClickHandler={rescanTrigger}
+            />
+          )}
+          <div className="scan-another">
+            <InternalLink
+              to="../"
+              onClick={() => gleanClick(`${OBSERVATORY}: click: scan-another`)}
+            >
+              Scan another website
+            </InternalLink>
+          </div>
+        </section>
+      </section>
+    </>
+  );
+}
+
+function hostAsRedirectChain(host, result: ObservatoryResult) {
+  const chain = result.tests.redirection?.route;
+  if (!chain || chain.length < 1) {
+    return host;
+  }
+  try {
+    const firstUrl = new URL(chain[0]);
+    const lastUrl = new URL(chain[chain.length - 1]);
+    if (firstUrl.hostname === lastUrl.hostname) {
+      return host;
+    }
+    return `${firstUrl.hostname} → ${lastUrl.hostname}`;
+  } catch (e) {
+    return host;
+  }
+}
+
+function CountdownButton({
+  host,
+  from,
+  duration,
+  title,
+  onClickHandler,
+}: {
+  host: string;
```

Comment:
Host is passed, but not used:
```suggestion
  from,
  duration,
  title,
  onClickHandler,
}: {
```